frontend/src/metabase/static-viz/components/PieChart/PieChart.tsx

```diff
@@ -38,11 +56,15 @@ export function PieChart({
 
   chart.setOption(option);
 
-  const svg = sanitizeSvgForBatik(chart.renderToSVGString());
+  const chartSvg = sanitizeSvgForBatik(chart.renderToSVGString());
 
   return (
-    <svg width={WIDTH} height={HEIGHT}>
-      <g dangerouslySetInnerHTML={{ __html: svg }}></g>
+    <svg width={WIDTH} height={PADDING_TOP + HEIGHT + legendHeight}>
+      <Legend />
```

Comment:
I believe the legend needs to be rendered below the chart like it previously was

Reply:
@kdoh can you please share your opinion on this? For line/area/bar/combo/row charts we render legends above the chart, but for Pie on dashboards it can be either on the left or below the chart. I think it makes sense to render the legend below the pie in the static viz

<img width="477" alt="Screenshot 2023-11-08 at 8 16 01 PM" src="https://github.com/metabase/metabase/assets/14301985/39d6929a-8c6a-4a4c-9fb4-dbb29748ca68">